Makefile

```
@@ -104,6 +104,10 @@ wakuexample2:
 	echo -e $(BUILD_MSG) "build/$@" && \
 		$(ENV_SCRIPT) nim wakuexample2 $(NIM_PARAMS) waku.nims
 
+wakuchat2:
```

```suggestion
chat2:
```

Might as well already go with the changes that are to come?

---

true